app/assets/stylesheets/description_box.scss

```diff
@@ -0,0 +1,8 @@
+.description {
+    border: 1px #333;
+    padding:5px;
+}
+.planet-text {
+    display:none;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered border, display<br>Colon after property should be followed by one space